skbio/stats/composition.py

```diff
@@ -608,10 +610,271 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0")
+def ancom(mat, cats,
+          alpha=0.05,
+          multicorr=False,
```

Comment:
I like `bonferroni`, but it is technically doing a different correction that plain Bonferroni, correct? 

Throwing in another name:
`correct_multiple_comparisons`

@jairideout Any thoughts? We will end up with this parameter again somewhere else undoubtedly.